include/effects/matrix/PatternPongClock.h

```diff
@@ -290,7 +290,7 @@ class PatternPongClock : public LEDStripEffect
         }
 
         // move bat 1 towards target
-        // if bat y greater than target y move down until hit 0 (dont go any further or bat will move off screen)
+        // if bat y greater than target y move down until hit 0 (don't go any further or bat will move off-screen)

```

Comment:
I argue that "off" and "screen" should not be connected with a hyphen, as it doesn't concern a compound adjective.

```suggestion
        // if bat y greater than target y move down until hit 0 (don't go any further or bat will move off screen)
```